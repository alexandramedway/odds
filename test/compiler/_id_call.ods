/* state PI */
/* state EUL */
state print(40 + 2)
state print("fourty-two")
state print(2 * (2 + 3))
