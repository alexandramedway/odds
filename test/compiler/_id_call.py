print((40 + 2))
print("fourty-two")
print((2 * (2 + 3)))
