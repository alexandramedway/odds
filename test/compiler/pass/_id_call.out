42
fourty-two
10
